backend/user_canister/src/canister.rs

```diff
@@ -201,18 +202,18 @@ impl Canister {
     }
 
     /// Upload file continue
-    pub fn upload_file_continue(request: UploadFileContinueRequest) {
+    pub fn upload_file_continue(request: UploadFileContinueRequest) -> UploadFileContinueResponse {
         let file = FileDataStorage::get_file(&request.file_id);
```

Comment:
rust trick: here you can do `let Some(file) = ...get_file(...) else { return ...; };`